frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -273,3 +273,22 @@ Feature: Topology chart area
               And user sees shortcut for Access create connector handle
               And user sees shortcut for Qpen quick search modal
               And user sees shortcut for Drag and drop a JAR file into Topology
+
+
+        @regression @manual @odc-5722
+        Scenario: Display of External Bindable resources: T-06-TC24
+            Given user has installed Service Binding operator
+              And user has created external bindable resource
```

Comment:
Can we add the steps to create external bindable resource with in this scenario, so that while executing this scenario. It is easier.
Include the test data as well

Reply:
We can only use kafka connection(RHOAS) as of now for the external bindable resource, I'll rewrite it as:
`user has created external bindable resource Kafka Connection "kafka-instance-123"`